### PR TITLE
Schemas

### PR DESCRIPTION
**Pull request by samuelcolvin**
implement #129

Sadly the syntax I has hoped to use (tuples of `default, Schema`) doesn't work since tuple packing with variable type annotations turns out to be a syntax error (referenced in parsing https://github.com/python/typing/issues/258) it works with parentheses but that's annoying.

Instead I've moved the default declaration into the schema

```py
class FoobarModel(BaseModel):
    foo: int = Schema(4, title='Foo Title')
    bar: str = Schema(..., description='this description of bar')
```

### CODE REVIEW

File: pydantic/fields.py [hunk 1]
```diff
@@ -32,11 +32,21 @@ class Validator(NamedTuple):
     check_fields: bool
 
 
+class Schema:
```
Comment (dsully) [inline pydantic/fields.py]: Adding a docstring and/or comment would be helpful here to know what this class is for.

Comment (samuelcolvin) [reply pydantic/fields.py]: will do, I need to do docs for this in general.

**Comment by samuelcolvin** [comment]
Needs docs, but otherwise think this is ready.

**Comment by Gr1N** [comment]
@samuelcolvin PR looks great! When you're plan to merge and release change?

**Comment by samuelcolvin** [comment]
Just needs docs. I'll do it soon.

**Pull request merged**